docs/web/examples/websocket/webskt.rpy

```diff
@@ -0,0 +1,43 @@
+from twisted.python.failure import Failure
+from twisted.web.iweb import IRequest
+from twisted.web.resource import Resource
+from twisted.web.static import File
+from twisted.web.websocket import WebSocketResource, WebSocketTransport
+from twisted.internet.task import LoopingCall
+
+
+class WebSocketDemo:
```

Comment:
I think that it would help to have some docstring inline.
```suggestion
class WebSocketDemo:
   """
   This is instantiated for each new websocket connection.
   """
```